tests/unittests/lib.collections.art

```diff
@@ -702,10 +702,21 @@ do [
 
     debug split [[1 2 3] [4 5 6] [7 8]]
 
-    ;path
-    ; temporarily disable
-    ; see issue: #872
-    ; debug split.path "directory/wofilerld"
+
+    debug split.path "directory\\wofilerld"
+    debug split.path "D:\\User\\user\\.arturo"
+    debug split.path "/usr/bin"
+
+    p1: "directory\\wofilerld"
+    p2: "D:\\User\\user\\.arturo"
+    p3: "/usr/bin"
```

Comment:
```suggestion
    (sys\os == "windows")? [
        debug split.path "directory\\wofilerld"
        debug split.path "\\usr\\bin"

        p1: "directory\\wofilerld"
        p2: "\\usr\\bin"
    ][
        debug split.path "directory/wofilerld"
        debug split.path "/usr/bin"

        p1: "directory/wofilerld"
        p2: "/usr/bin"
    ]
```

Basically, we could use different paths based on the active OS (and I have removed the ones that wouldn't look very normal for Linux/macOS so that - if we insist on testing common output - the output can possibly be the same on all OSes...)

Reply:
Yes, I agree. But, since Windows uses the alt `/` too, I think it should be covered.